pkg/migrations/sql/idempotent/base.sql

```diff
@@ -1163,12 +1163,6 @@ BEGIN
    SELECT mtn.id, mtn.table_name FROM SCHEMA_CATALOG.get_or_create_metric_table_name(metric_name) mtn
```

Comment:
@antekresic for future optimization: we should be able to get rid of this call too if we cache the metric_id and table_name in the connector code. That would optimize this path further.